ffi/framebuffer_android.lua

```diff
@@ -149,35 +159,55 @@ end
 function framebuffer:refreshPartialImp(x, y, w, h)
     self:_updateWindow()
     if has_eink_full_support then
-        self:_updatePartial(partial_regal, 0, x, y, w, h)
+        if eink_platform == "qualcomm" then
```

Comment:
These won't happen unless ONYX_2 is `HAS_FULL_EINK_SUPPORT` in luajit-launcher

Reply:
It is, I assume, `HAS_FULL_EINK_SUPPORT` means device is capable of partial updates. If that's the case then Nova 2 is capable of that.

Reply:
Duh, my bad. I was under the impression you removed HAS_FULL_EINK_SUPPORT in DeviceInfo.kt. It's not the case :p 

Apologies!